cincinnati/src/plugins/internal/graph_builder/openshift_secondary_metadata_parser/plugin.rs

```diff
@@ -295,6 +295,26 @@ impl OpenshiftSecondaryMetadataParserPlugin {
         }
     }
 
+    fn process_version(
+        &self,
+        data_dir: &PathBuf,
+    ) -> Fallible<()> {
+        let supported_version = ["1.0.0"];
```

Comment:
This array should not be local to this function. This was it looks like the version is hard coded.  The variable name should be metadata_schema_version.

Reply:
> This array should not be local to this function.

I would like to keep the set of supported versions internal, because I do not consider it a public API.  Folks who want to know if a graph-data checkout is supported should just load it with this plugin, and report back any error messages they see.  More on whether that's a viable option today vs. the current CI executable in [this thread][1], so I'm closing this to consolidate discussion there.

[1]: https://github.com/openshift/cincinnati/pull/314#discussion_r499784365